spec/solver_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rspec'
+require './solver'
+
+describe Solver do
+  describe '#factorial' do
+    it 'returns factorial of a given number' do
+      expect(subject.factorial(5)).to eql(120)
+    end
+  end
```

Comment:
- Great Job for testing Positive value :+1: I think it would be better if you have tests for all cases in TDD here for factorial you cab add tests for `Negative value` and `0 value` which will return 0

Reply:
Hello @binhussen ,

Thanks a lot for the review. However, I think the factorial of `0 value` returns `1`

Thanks